src/Picqer/Financials/Exact/StockPosition.php

```diff
@@ -1,39 +1,33 @@
-<?php
-namespace Picqer\Financials\Exact;
+<?php namespace Picqer\Financials\Exact;
 
 /**
- * Entity holding stock position details.
+ * Class StockPosition
  *
  * @package Picqer\Financials\Exact
  * @see https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=ReadLogisticsStockPosition
- *
- * @property $InStock
- * @property $ItemId
- * @property $PlanningIn
- * @property $PlanningOut
+ * 
+ * @property Double $InStock Number of items in stock
+ * @property Guid $ItemId Primary key, Item
+ * @property Double $PlanningIn Number of items that are planned to come in
+ * @property Double $PlanningOut Number of items that are planned to go out
  */
 class StockPosition extends Model
 {
     use Query\Findable;
+    use Query\Relatable;
 
     /**
-     * The fillable properties for the StockPosition model.
-     *
-     * @var string[]
+     * @var string Name of the parent key for this model
      */
-    protected $fillable = ['InStock', 'ItemId', 'PlanningIn', 'PlanningOut'];
+    protected $parentKey = 'itemId';
+
+    protected $fillable = [
+        'InStock',
+        'ItemId',
+        'PlanningIn',
+        'PlanningOut'
+    ];
 
-    /**
-     * The API request URL slug.
-     *
-     * @var string
-     */
     protected $url = 'read/logistics/StockPosition';
 
-    /**
-     * The primary key for the current entity.
-     *
-     * @var string
-     */
-    protected $primaryKey = 'ItemId';
```

Comment:
Hmm, this is a function and therefor doesn't have a primary key. (There can be many rows of stockPosition having the same ItemId.) What shall we do?